.github/workflows/build.yml

```diff
@@ -11,7 +11,7 @@ jobs:
   build:
     strategy:
       matrix:
-        go-version: [1.14.x, 1.15.x]
+        go-version: [1.18.x]
```

Comment:
if we bump this, we also want to bump in go.mod here: https://github.com/expel-io/aws-resource-counter/blob/main/go.mod#L3 and probably test that the builds end up working as we'd expect

Reply:
i dont know if you have an arm chip machine, but i know a few expel engineers have one, we can ask them to try it